app/lib/nomis_client/court_cases.rb

```diff
@@ -3,13 +3,21 @@
 module NomisClient
   class CourtCases < NomisClient::Base
     class << self
-      ACTIVE_CASES_FILTER = 'activeOnly=true'
+      def get(booking_id, filter_params)
```

Comment:
Maybe you can add a default value to the filter, so you can avoid to specify one.